mlir/lib/CAPI/IR/IR.cpp

```diff
@@ -717,17 +717,34 @@ void mlirOperationMoveBefore(MlirOperation op, MlirOperation other) {
   return unwrap(op)->moveBefore(unwrap(other));
 }
 
+static mlir::WalkResult translateWalkResult(MlirWalkResult result) {
```

Comment:
```suggestion
static mlir::WalkResult unwrap(MlirWalkResult result) {
```

this is the convention we use for all other types.